common/interfaces/desktop_app_interface/usb_internals.c

```diff
@@ -303,25 +304,31 @@ static comm_error_code_t comm_process_abort_packet(const packet_t *rx_packet) {
  */
 static void send_status_packet(const packet_t *rx_packet) {
   uint8_t payload[COMM_MAX_PAYLOAD_SIZE];
```

Comment:
Since we are only sending one chunk out of one total chunk (call to `comm_write_packet`), therefore, we are fixing the length of data to be sent via this function.

Consider a case, wherein protobuf encoding passed due to the `6KiB` buffer size, but it will not be easily detected since the physical layer will only transfer 1 packet of data. 

So consider reducing size of `payload` with appropriate size

Reply:
This was my wrong understanding regarding `COMM_MAX_PAYLOAD_SIZE`. I got confused and thought this was a 6KiB buffer size.

This comment is not valid anymore, so closing the conversation.

Reply:
the size of `payload` is 48.

it is not  `6KiB`